### PR TITLE
Fix: Hex values can now be prefixed with a '#'

### LINKED ISSUE #215: [ColourRole] Hexcode with # prefix doesn't work
### Describe the bug
ColourRole doesn't recognise hexcodes beginning with #
image

### To Reproduce
1. `k!enableExt ColourRole`
2. Give yourself a role with customColour allowed
3. `k!customColour #52E91E`
4. See error (Invalid colour string specified, make sure it's a valid colour hex.)

### Expected behaviour
image

### Screenshots
N/A

### Additional context
N/A

### PR DESCRIPTION
**Pull request by sudoPom**
## Summary

 `close #215 `
## Checklist

- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

<br>

- [x] Have you tested the changes? ([pytest](https://docs.pytest.org/) & [dpytest](https://dpytest.readthedocs.io/))
- [x] Have you followed [PEP-8](https://www.python.org/dev/peps/pep-0008/) for naming and styling?  
- [x] Has your code been properly documented with RestructuredText docstrings?
- [x] Have you added your changes to `CHANGELOG.md` under the `[Unreleased]` heading?
- [ ] If your code added new bot commands, have you updated `documentation.json`?

<br>

- [x] All of this code is your own, or follows the author repo's licence.

### CODE REVIEW

**Comment by Kaspiaan** [comment]
Please add the new additions to the existing tests, you want to make sure the test code coverage is as high as possible.

**Comment by JayDwee** [comment]
Also please add all changes including bug fixes to the changelog as mentioned in the checklist

**Comment by JayDwee** [comment]
Remember, when you think you are done request a review from Kieran, and/or Viraj 
image

**Comment by Kaspiaan** [comment]
Done a review and all looks ok, going to request VIraj also have a look over

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ A lot of these commands will only be available to administrators
 ## [Unreleased]
 ### Other
 - Testing updated to use builders in dpytest 0.5.0
+- Colour strings can now be prefixed with a #
```
Comment (Kaspiaan) [inline CHANGELOG.md:8]: Can you put this under Colour Role please (See previous changelogs)

**Review by Kaspiaan — CHANGES_REQUESTED** [review]
Just a tiny change :)

**Review by Kaspiaan — APPROVED** [review]

**Pull request merged**